editor/components/block-list/index.js

```diff
@@ -245,10 +246,10 @@ class BlockList extends Component {
 	}
 
 	render() {
-		const { blocks, showContextualToolbar } = this.props;
+		const { blocks, showContextualToolbar, renderBlockMenu } = this.props;
 
 		return (
-			<div>
+			<BlockSelectionClearer>
```

Comment:
Why was this changed from a `div` to `BlockSelectionClearer` ?

Reply:
Because this was handled in the parent component by using a ref to this div there. I just made this component generic and used it in both the `VisualEditor` component and the inner component `BlockList`